diff --git a/dist/commands-D4xZQ5t-.mjs b/dist/commands-D4xZQ5t-.mjs
index 154d99547dd613ce4f046e572ac71c1ab9a663e2..e575c708b0e9664c4134e6409b55cd003a96920c 100644
--- a/dist/commands-D4xZQ5t-.mjs
+++ b/dist/commands-D4xZQ5t-.mjs
@@ -2930,2 +2930,5 @@
 async function readWorkspaceVersions(cwd) {
-	const packages = await getWorkspacePackages(cwd);
+	const packages = await getWorkspacePackages(cwd, {
+		ignoreRootPackage: false,
+		ignorePrivatePackage: false
+	});
