diff --git a/dist/node/chunks/dep-DDxXL6bt.js b/dist/node/chunks/dep-DDxXL6bt.js
index 1d5edd336a5216bd7b465392727d1434a97cfcb4..bd505effcd53ae1d74f3a70ac53621d14a24454c 100644
--- a/dist/node/chunks/dep-DDxXL6bt.js
+++ b/dist/node/chunks/dep-DDxXL6bt.js
@@ -48909,7 +48909,7 @@ function createCSSResolvers(config) {
   return {
     get css() {
       return cssResolve ??= createBackCompatIdResolver(config, {
-        extensions: [".css"],
+        extensions: [".css",".wxss"],
         mainFields: ["style"],
         conditions: ["style", DEV_PROD_CONDITION],
         tryIndex: false,
@@ -48919,7 +48919,7 @@ function createCSSResolvers(config) {
     get sass() {
       if (!sassResolve) {
         const resolver = createBackCompatIdResolver(config, {
-          extensions: [".scss", ".sass", ".css"],
+          extensions: [".scss", ".sass", ".css",".wxss"],
           mainFields: ["sass", "style"],
           conditions: ["sass", "style", DEV_PROD_CONDITION],
           tryIndex: true,
@@ -48942,7 +48942,7 @@ function createCSSResolvers(config) {
     },
     get less() {
       return lessResolve ??= createBackCompatIdResolver(config, {
-        extensions: [".less", ".css"],
+        extensions: [".less", ".css",".wxss"],
         mainFields: ["less", "style"],
         conditions: ["less", "style", DEV_PROD_CONDITION],
         tryIndex: false,
